eth/backend.go

```diff
@@ -808,9 +808,16 @@ func (s *Ethereum) Start() error {
 		}(i)
 	}
 	time.Sleep(10 * time.Millisecond) // just to reduce logs order confusion
-
-	go stages2.StageLoop(s.sentryCtx, s.chainDB, s.stagedSync, s.sentriesClient.Hd, s.notifications, s.sentriesClient.UpdateHead, s.waitForStageLoopStop, s.config.SyncLoopThrottle)
-
+	go func() {
```

Comment:
Is this where the wait for sync start is done if it fails?

Reply:
The error for retry is inside of the stage loop function.